org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/logger/LogbackLoggerSupport.java

```diff
@@ -9,22 +9,51 @@
  */
 package org.eclipse.scout.rt.platform.logger;
 
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.eclipse.scout.rt.platform.util.Assertions;
 import org.slf4j.ILoggerFactory;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import ch.qos.logback.classic.Level;
 import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.core.spi.ConfigurationEvent.EventType;
 
 /**
+ * <p>
  * Logger support for Logback.
+ * </p>
+ * <p>
+ * With support for change tracking to automatically re-apply all changes after Logback reloads its logging
+ * configuration (e.g. configured by periodic rescan)
+ * </p>
  *
  * @since 5.2
  */
 public class LogbackLoggerSupport extends AbstractLoggerSupport {
 
   private static final Logger LOG = LoggerFactory.getLogger(LogbackLoggerSupport.class);
 
+  private volatile Map<Logger, Optional<LogLevel>> m_changeMap = new ConcurrentHashMap<>();
```

Comment:
Add javadoc like "map caching log levels for loggers whose level was changed dynamically after initialization"

Reply:
Fixed in 6e0a26a3aa4980807138223d90d4956d76d2a13a